libs/ui/src/precinct_scanner_tally_qrcode.tsx

```diff
@@ -1,82 +1,69 @@
-import { CompressedTally, ElectionDefinition } from '@votingworks/types';
+import {
+  CompressedTally,
+  Election,
+  ElectionDefinition,
+} from '@votingworks/types';
 import { format, formatFullDateTimeZone } from '@votingworks/utils';
 import { DateTime } from 'luxon';
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import { LogoMark } from './logo_mark';
 import { Prose } from './prose';
 import { ReportSection } from './tally_report';
 import { Text } from './text';
 import { QrCode } from './qrcode';
 
-const QrCodeWrapper = styled.div`
-  width: 25%;
-`;
-
-interface Props {
-  pollsToggledTime: number;
+interface SignedQuickResultsReportingUrlProps {
   electionDefinition: ElectionDefinition;
   signingMachineId: string;
-  isPollsOpen: boolean;
   isLiveMode: boolean;
   compressedTally: CompressedTally;
 }
 
-export function PrecinctScannerTallyQrCode({
-  pollsToggledTime,
+export async function getSignedQuickResultsReportingUrl({
```

Comment:
Taking this out of the component itself, expecting it to be called elsewhere. Where possible, I think we should avoid effects in printed components. Otherwise we would have waited on this to render with an `onRendered` callback with the new printing approach. That may be easier but presents more footguns later, I think.